protos/fpc/fpc.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package fpc;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CC_Parameters {
+    string chaincode_id   = 1;      // chaincode name
+    string version        = 2;      // chaincode version, this is the expected mrenclave
```

Comment:
We probably should mention here (and for tlcc_mrenclave) what encoding happens (or just link to fpc-management.md for version and mention that tlcc_mrenclave is handled the same way? Couldn't find a definition on encoding of tlcc_mrenclave in the .md ...)

This more or less applies also to most other `bytes` where the encoding is not necessarily clear. (As discussed on phone, it might be worthwhile also to define them as types (sub-messages) if there is a good way to augment the protoc/nanopb-generated code with corresponding encoding/decoding functions ...

Reply:
in `fpc-management.md` we define:
```
* `--version <MRENCLAVE as hexadecimal string>`, the version of the FPC chaincode must contain a string that represent the identity of the enclave. This is the same identity which the trusted hardware computes and attests to. The string can be conveniently found in the `mrenclave` output file of the `generate_mrenclave.sh` script.
```

will add a comment here

Reply:
that's the reference to version encoding i also referenced; what i meant it seems we haven't done the equivalent for tlcc anywhere (but naturally might use the same encoding, even though for this case the original binaries might work as well for the comparison, but then for debugging/logging makes sense to use same encoding ..)